MH-FiorellaTheme5/source/index.html.erb

```diff
@@ -1,17 +1,13 @@
----
-title: Welcome to Middleman
----
-<!--
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 340" class="middleman-logo" aria-labelledby="middleman-logo__title" role="img">
-  <title id="middleman-logo__title">Middleman</title>
-  <path class="middleman-logo__top-left-bar" fill-opacity=".45" d="M0 40L200 0v30L0 60z"/>
-  <path class="middleman-logo__top-right-bar" fill="#fff" d="M200 0l200 40v20L200 30z"/>
-  <path class="middleman-logo__left-m" fill-opacity=".45" d="M0 78v184l45 5V152l45 83 47-83v129l53 7V52l-57 8-43 83-43-70z"/>
-  <path class="middleman-logo__right-m" fill="#fff" d="M400 78v184l-45 5V152l-45 83-47-83v129l-53 7V52l57 8 43 83 43-70z"/>
-  <path class="middleman-logo__bottom-left-bar" fill-opacity=".45" d="M0 300l200 40v-30L0 280z"/>
-  <path class="middleman-logo__bottom-right-bar" fill="#fff" d="M200 340l200-40v-20l-200 30z"/>
-</svg>
+<div class="container">
+  <%= partial "partials/_slide-out-menu" %>
 
-<h1>
-  Middleman is Running
-</h1>
+  <%= partial "partials/_header" %>
```

Comment:
You don't need to add the `_` before the partial name. The partial gets detected just by using the name, no need to use the underscore. Can you please remove all of them from the partial rendering. 👍